### PR TITLE
Cert manager v3.3.0

### PR DESCRIPTION
**Pull request by ubergesundheit**
Combination of #149 #148 plus enabling of ciliumNetworkPolicies

### Checklist

- [ ] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

**Review by fiunchinho — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 - Set `external-dns` config for upgrade to v3.
+- Set `chart-operator.giantswarm.io/force-helm-upgrade` annotation to `false` for `cert-manager` App CR.
```
Comment (ubergesundheit) [inline CHANGELOG.md:13]:
```suggestion
- Set `chart-operator.giantswarm.io/force-helm-upgrade` annotation to `false` for `cert-manager` App CR.
- Enable cilium NetworkPolicies for `cert-manager`.
```

**Comment by AverageMarcus** [comment]
Looks like there was an error connecting to GitHub 😞  Retrying...

/run cluster-test-suites

**Pull request merged**